eopdev/ap-southeast-2/eopdev/services/ecs-eop-ingest-api/module_config.hcl

```diff
@@ -0,0 +1,7 @@
+# Define some config vars that can be imported by the shared terragrunt config. To keep the config dry.
+locals {
+  config_secrets_manager_arn = "arn:aws:secretsmanager:ap-southeast-2:657968434173:secret:EOPIngestAPIConfig-Sd0J4T"
```

Comment:
This currently has values for CONFIG_KEYSTORE_CONTENT, CONFIG_KEYSTORE_PASSWORD and CONFIG_KEYSTORE_KEY copied from the `EOPManagerConfig` secret.